portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -92,12 +126,12 @@ private String convertListToJson(ArrayList<Comment> comments) {
    * @return the request parameter, or the default value if the parameter
    *         was not specified by the client
    */
-  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
-    String value = request.getParameter(name);
-    if (value == null) {
-      return defaultValue;
+    private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
I'd suggest using Optionals here. This method can return an Optional<String> and then the caller can use the method Optional.orElse("5") to use a default value

https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html

Reply:
I tried implementing with Optionals. Let me know what you think.

Reply:
Nice! LGTM